### PR TITLE
Update support's packages version

### PR DESCRIPTION
**Pull request by AnastasiaKubova**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] ~Are the files formatted correctly?~
* [ ] ~Did you add unit tests if this modifies the Windows code?~
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Updated support's packages version.

## Related PRs or issues

[AB#70915](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/70915)

### CODE REVIEW

File: SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/packages.config [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  <package id="Xamarin.Android.Support.Vector.Drawable" version="28.0.0.1" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Arch.Core.Common" version="1.1.1.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Arch.Lifecycle.Common" version="1.1.1.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Arch.Lifecycle.Runtime" version="1.1.1.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.Animated.Vector.Drawable" version="28.0.0.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.Annotations" version="28.0.0.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.Compat" version="28.0.0.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.Core.UI" version="28.0.0.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.Core.Utils" version="28.0.0.3" targetFramework="monoandroid81" />
+  <package id="Xamarin.Android.Support.CustomTabs" version="28.0.0.3" targetFramework="monoandroid81" />
```
Comment (guperrot) [inline SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/packages.config]: don't we need to retarget to monoandroid90 here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 2.5.1 （Under development）
 
+### App Center
+
+#### Xamarin
+
+* **[Fix]** Updated Android support packages to 28.0.0.3 version in Auth and Push module.
```
Comment (MatkovIvan) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Updated Android support packages to `28.0.0.3` version in Auth and Push modules.
```

File: SDK/AppCenterPush/Microsoft.AppCenter.Push.Android/Microsoft.AppCenter.Push.Android.csproj [hunk 3]
```diff
@@ -103,16 +103,38 @@
...
   <Import Project="..\..\..\packages\Xamarin.Firebase.Common.71.1610.0\build\MonoAndroid90\Xamarin.Firebase.Common.targets" Condition="Exists('..\..\..\packages\Xamarin.Firebase.Common.71.1610.0\build\MonoAndroid90\Xamarin.Firebase.Common.targets')" />
   <Import Project="..\..\..\packages\Xamarin.Firebase.Iid.71.1710.0\build\MonoAndroid90\Xamarin.Firebase.Iid.targets" Condition="Exists('..\..\..\packages\Xamarin.Firebase.Iid.71.1710.0\build\MonoAndroid90\Xamarin.Firebase.Iid.targets')" />
   <Import Project="..\..\..\packages\Xamarin.Firebase.Messaging.71.1740.0\build\MonoAndroid90\Xamarin.Firebase.Messaging.targets" Condition="Exists('..\..\..\packages\Xamarin.Firebase.Messaging.71.1740.0\build\MonoAndroid90\Xamarin.Firebase.Messaging.targets')" />
+  <Import Project="..\..\..\packages\Xamarin.Android.Support.Annotations.28.0.0.3\build\monoandroid90\Xamarin.Android.Support.Annotations.targets" Condition="Exists('..\..\..\packages\Xamarin.Android.Support.Annotations.28.0.0.3\build\monoandroid90\Xamarin.Android.Support.Annotations.targets')" />
```
Comment (jwallra) [inline SDK/AppCenterPush/Microsoft.AppCenter.Push.Android/Microsoft.AppCenter.Push.Android.csproj]: Why are so many imports added to this csproj and the corresponding package.config?  We should remove them if they're not needed.

Comment (guperrot) [reply SDK/AppCenterPush/Microsoft.AppCenter.Push.Android/Microsoft.AppCenter.Push.Android.csproj]: It's pulled by dependencies of dependencies unfortunately.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 2.5.1 （Under development）
 
+### App Center
+
+#### Xamarin
+
+* **[Fix]** Updated Android support packages to `28.0.0.3` version in Auth and Push modules.
```
Comment (Jamminroot) [inline CHANGELOG.md]:
```suggestion
* **[Fix]** Updated Android support packages version to `28.0.0.3` for Auth and Push modules.
```

**Comment by guperrot** [comment]
After more testing in forms puppet, forms demo (both with the nugets) and even a new blank app, the best way to solve conflicts is to remove the old workaround for helping conflicts.

Trying to set the support libs as dependencies in the nuspec is now causing more problems than the ones we originally solved in the past. With recent Xamarin.Forms versions I could only make this work by simply removing all the support dependencies from nuspec.

**Comment by guperrot** [comment]
Ok except we need to keep custom tabs for non forms... (xamarin forms is the one depending on custom tabs) will try to fix just push then.

**Comment by guperrot** [comment]
I cannot repro https://github.com/microsoft/appcenter-sdk-dotnet/pull/1188#pullrequestreview-313610324 must have been something weird I did? I won't block anymore I guess.

**Pull request merged**